include/openmc/distribution.h

```diff
@@ -48,13 +48,19 @@ class Discrete : public Distribution {
   // Properties
   const vector<double>& x() const { return x_; }
   const vector<double>& p() const { return p_; }
+  const vector<size_t>& alias() const { return alias_; }
 
 private:
   vector<double> x_; //!< Possible outcomes
-  vector<double> p_; //!< Probability of each outcome
+  vector<double>
+    p_; //!< Probability of each outcome, mapped to alias method table
```

Comment:
With the alias method, this is not really the probably of the outcome anymore (what was originally passed as `p`). I would suggest changing this to `prob_` to avoid confusion and make clear that it is the probability of accepting the uniformly sampled bin.

Along similar lines, the `p()` method above won't really make sense to a user of this class -- I would suggest we remove it altogether (doesn't look like it's actually used anywhere).